### ISSUE TITLE
Import bookmarks from other browsers and HTML file.

### ISSUE THREAD

**Issue by s8321414** [OtterBrowser/otter-browser#1114]
As title says, we need this feature! :smiley:

**Comment by Chocimier** [comment]
You can already import html bookmarks and even more data from Opera under Settings → Import and export menu.

**Comment by ZarkBit** [comment]
![z0t 1](https://cloud.githubusercontent.com/assets/4160341/14296426/f343a0ce-fb71-11e5-83c9-295a0e9dd04e.jpg)

**Comment by s8321414** [comment]
But can we import bookmarks from other browsers like Chrome (Chromium), Opera (Blink version), Firefox, etc.?

**Comment by Zero3K** [comment]
No, but I agree with being able to import bookmarks and sessions from browsers other than Opera 12/Netscape.

**Comment by s8321414** [comment]
Yeah, so I think it's good feature :smile:

**Comment by Frenzie** [comment]
If you first export their bookmarks as HTML. (Well, I don't know if
Chromium and derivatives can do that, but if not I'm sure there's an
extension for it.)

**Comment by The-Compiler** [comment]
Chromium/Firefox/IE all support exporting bookmarks in the Netscape HTML format. If Otter imports that, that should just work(tm).

**Comment by pierreporte** [comment]
Would it be possible to import the bookmarks automatically? For instance, you have a wizard that detects all browser installed of the system. The user selects the browser he or she wants to import bookmarks from and its done. It’s not a request, just a question.

**Comment by Frenzie** [comment]
Possible — to a certain extent, sure. But I think it sounds like an awful
lot of work for precariously little gain. :)

**Comment by s8321414** [comment]
hmmm... I prefer importing bookmarks from other browsers directly...

**Comment by Emdek** [comment]
Chrome / Chromium and their forks should be easy to handle:
http://fileformats.archiveteam.org/wiki/Chrome_bookmarks
While Firefox abuses SQLite for everything, and I would prefer to not readd dependency on QtSql module.
